legacy-buildtools/README.md

```diff
@@ -0,0 +1,29 @@
+# Support for the Legacy Libyui CMake Environment
+
+The contents of this directory are installed to /usr/share/libyui/buildtools to
+support existing community-supported libyui plugins and extensions.
+
+This used to be the starting point of the prevous libyui build system where a
```

Comment:
Typo: prevous.

Reply:
Fixed the typo.

As for separate -doc.spec files: I am not sure. Actually, the doc packages don't have very much in common with their main packages; very different BuildRequires, different build and install sections.

Actually, I'd very much prefer to drop those libyui*-doc packages completely. They almost exclusively consist of doxygen-generated autodocs. Those autodocs offer little advantage over simply reading the header files; some nice formatting, yes, some class diagrams, some hyperlinks to their base classes.

But anybody developing an application using libyui or a libyui extension will almost certainly want to clone the libyui repos anyway: To get a better picture what it's all about, to be able to use `grep`, to be sure what exactly happens in each possible situation.

And anybody who has such a git clone can easily generate the autodocs and even read them in-place (no need for `make install-doc`).

So, instead of micro-optimizing those libyui*-doc.spec files, I'd strongly vote for doing a clean cut and getting rid of them completely. They serve no purpose, they are just a constant hassle.

Reply:
I'd like to unify the VERSIONS.cmake files into a central one once we merged the libyui repos.

As for the version numbers in the .spec files, that would mean going back to generating them from a .spec.in each. We had that in the past, and for some reason (which I don't know) it was decided to no longer do that.